### PR TITLE
Add node v4 test configuration (close #102)

### LINKED ISSUE #102: Add node v4 test configuration

### PR DESCRIPTION
**Pull request by churkin**
\cc @AlexanderMoskovkin @inikulin

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 language: node_js
 matrix:
  include:
+   - node_js: "4.1"
+     env: GULP_TASK="server-tests"
    - node_js: "0.10"
      env: GULP_TASK="server-tests"
    - node_js: "0.12"
```
Comment (inikulin) [inline .travis.yml]: We can remov `0.12` now

**Comment by inikulin** [comment]
\r-

**Comment by inikulin** [comment]
@churkin We can replace
```
node_js: 4.1
```
with
```
node_js: stable
```
So it will always peek latest stable version. Sorry for the late call.

**Comment by churkin** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by inikulin** [comment]
lgtm

**Pull request merged**